mozilla-release/ipc/app/module.ver

```diff
@@ -4,3 +4,5 @@ WIN32_MODULE_PRODUCTVERSION_STRING=@MOZ_APP_VERSION@
 WIN32_MODULE_DESCRIPTION=Plugin Container for @MOZ_APP_DISPLAYNAME@
 WIN32_MODULE_PRODUCTNAME=@MOZ_APP_DISPLAYNAME@
 WIN32_MODULE_NAME=@MOZ_APP_DISPLAYNAME@
+WIN32_MODULE_FILEVERSION=@MOZ_APP_WINVERSION@

```

Comment:
I don't know yet. Module.ver are responsible for adding information into RC file on windows build, so later, on linking, this information will be placed into binary files. Here we replace default FF version value to our own version (i did similar for all module.ver file for consistent). For example this module.ver used in building plugin-container.exe